### PR TITLE
[Merged by Bors] - feat(Order/InitialSeg): extra lemmas on `α ≤i β`

### PR DESCRIPTION
**Pull request by vihdzp**
---

- [x] depends on: #17678

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by YaelDillies — COMMENTED** [review]
Thanks! 🚀

maintainer merge

File: Mathlib/Order/InitialSeg.lean [hunk 1]
```diff
@@ -6,6 +6,7 @@ Authors: Mario Carneiro, Floris van Doorn, Violeta Hernández Palacios
 import Mathlib.Data.Sum.Order
 import Mathlib.Logic.Equiv.Set
 import Mathlib.Order.RelIso.Set
+import Mathlib.Order.UpperLower.Defs
```
Comment (urkud) [inline Mathlib/Order/InitialSeg.lean:9]:
```suggestion
```

**Comment by urkud** [comment]
Thanks! Once CI is happy, please merge this PR using `bors r+`
bors d+

**Pull request closed without merging**